### PR TITLE
Upgrading puppet stdlib using puppet module

### PR DESCRIPTION
**Pull request by jmbabazi**

### CODE REVIEW

File: gem-update.sh [hunk 1]
```diff
@@ -31,5 +31,7 @@ librarian-puppet install
 
 fi
 
+puppet module upgrade puppetlabs-stdlib --version 6.1.0 --modulepath=/etc/puppet/modules
```
Comment (brandones) [inline gem-update.sh]: Based on [the compatibility sheet](https://forge.puppet.com/puppetlabs/stdlib/compatibility), this version of stdlib shouldn't be compatible with puppet below version  5.5.10. What's the version of puppet you checked it with? Are you able to try this on a computer with puppet 3.x applied?

Comment (jmbabazi) [reply gem-update.sh]: Thanks @brandones , this seems to work on puppet 4.10.10 which is what we use on Ubuntu 16. I think I am going to change to version 4.25.1(which still doesn't work on 14.04 but the overall idea is to completely move away from ubuntu 14 and puppet 3...)

**Comment by brandones** [comment]
Merge whenever, and then please ping me so that I can get https://pihemr.atlassian.net/browse/MEX-129 back in.

File: Puppetfile.lock [hunk 2]
```diff
@@ -2,7 +2,7 @@ FORGE
   remote: https://forgeapi.puppetlabs.com
   specs:
     maestrodev-wget (1.7.0)
-    puppetlabs-stdlib (4.12.0)
+    puppetlabs-stdlib (4.25.1)
     puppetlabs-vcsrepo (1.3.1)
```
Comment (mogoodrich) [inline Puppetfile.lock]: I think technically we are supposed to update the Puppetfile as well, but not sure.. but, yeah, let's just go ahead with this for now.

File: gem-update.sh [hunk 3]
```diff
@@ -31,5 +31,8 @@ librarian-puppet install
 
 fi
 
+#this is include in Puppetfile.lock
+#puppet module upgrade puppetlabs-stdlib --version 6.1.0 --modulepath=/etc/puppet/modules
+
```
Comment (mogoodrich) [inline gem-update.sh]: let's just delete this, as we will forget what is does in the future!

**Pull request merged**